src/dosbox.cpp

```diff
@@ -1331,8 +1331,8 @@ void DOSBOX_Init()
 	control->AddSection_line("autoexec", &AUTOEXEC_Init);
 
 	MSG_Add("AUTOEXEC_CONFIGFILE_HELP",
-	        "Lines in this section will be run at startup.\n"
-	        "You can put your MOUNT lines here.\n");
+	        "Every line in this section is executed at startup as DOS commands.\n"
```

Comment:
Maybe:

_Every line in this section is executed at startup as a DOS command._

or

_All the lines in this section are executed at startup as DOS commands._

Reply:
Nice catch, I'll take the first 😄